tests/simulator/initialize/test_fields_init_2d.py

```diff
@@ -34,10 +34,7 @@ def test_density_is_as_provided_by_user(self, interp_order):
         print(f"\n{self._testMethodName}_{ndim}d")
         self._test_density_is_as_provided_by_user(ndim, interp_order)
 
-    @data(*interp_orders)
-    def test_density_decreases_as_1overSqrtN(self, interp_order):
-        print(f"\n{self._testMethodName}_{ndim}d")
-        self._test_density_decreases_as_1overSqrtN(ndim, interp_order)
+
```

Comment:
what's the reason already, taking too long?

Reply:
it was only ever setup for 1d, and the 2d test just ran it in 1d again